src/queries/catalog/linter-query/linter-query-executor.ts

```diff
@@ -0,0 +1,29 @@
+import type { BasicQueryData } from '../../base-query-format';
+import type { LinterQuery, LinterQueryResult } from './linter-query-format';
+import { runSearch } from '../../../search/flowr-search-executor';
+import { FlowrSearchElements } from '../../../search/flowr-search';
+import type { LintingRuleNames, LintingRuleResult } from '../../../linter/linter-rules';
+import { LintingRules } from '../../../linter/linter-rules';
+
+
+export function executeLinterQuery({ ast, dataflow }: BasicQueryData, queries: readonly LinterQuery[]): LinterQueryResult {
```

Comment:
We should probably add warnings similar to other queries if we query the same config multiple times